### PR TITLE
ssvlib: update request-form and footer-form

### PR DESCRIPTION
**Pull request by thanhtung2705**

### CODE REVIEW

File: ssvlibraries/pattern-lab/source/_patterns/01-components/form/footer-form.twig [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <div class="form-item">
+      <input type="text" placeholder="full name">
+    </div>
+
+    <div class="form-item">
+      <input type="email" placeholder="email">
+      <button type="submit">button</button>
+    </div>
+  </form>
+</div>
```
Comment (bienvu) [inline ssvlibraries/pattern-lab/source/_patterns/01-components/form/footer-form.twig]: missing line at bottom

File: ssvlibraries/pattern-lab/source/_patterns/01-components/form/footer-form.twig [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="footer-form">
```
Comment (bienvu) [inline ssvlibraries/pattern-lab/source/_patterns/01-components/form/footer-form.twig]: Change footer-form to box-form

File: ssvlibraries/scss/_styles-main.scss [hunk 3]
```diff
@@ -36,7 +36,8 @@
 //@import 'vendors/chosen';
 
 // Components
-// @import 'components/block-example';
+@import 'components/form';
+@import 'components/footer-form';
```
Comment (bienvu) [inline ssvlibraries/scss/_styles-main.scss]: change to box-form and add request form code to one file

File: ssvlibraries/scss/components/footer-form.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.footer-form {
+  padding: 30px 0;
+
+  .form-item {
+    &:last-of-type {
+      display: flex;
+    }
+  }
+}
```
Comment (bienvu) [inline ssvlibraries/scss/components/footer-form.scss]: missing bottom line

File: ssvlibraries/scss/components/form.scss [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+  &__title {
+    text-align: center;
+  }
+
+  .form-list {
+    flex-wrap: wrap;
+    margin: 0;
+
+    @media only screen and (min-width: 768px) {
```
Comment (bienvu) [inline ssvlibraries/scss/components/form.scss]: please use variable

File: ssvlibraries/scss/components/form.scss [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      justify-content: space-between;
+      margin: 0 -10px;
+      flex-wrap: nowrap;
+    }
+  }
+
+  .form-item {
+    margin: 0 0 10px;
+
+    @media only screen and (min-width: 768px) {
```
Comment (bienvu) [inline ssvlibraries/scss/components/form.scss]: please use variable

**Comment by lehung1109** [comment]
box-form  has been created

**Pull request closed without merging**